docs/guides/operator-guide-azure-enclave.md

```diff
@@ -9,129 +9,310 @@ sidebar_position: 18
+<!-- 
+* [Prerequisites](#prerequisites)
+   - [Set Up UID2 Operator Account](#set-up-uid2-operator-account)
+   - [Install Azure CLI](#install-azure-cli)
+   - [Get the Required Azure Permissions](#install-azure-cli)
+* [Deployment Environments](#deployment-environments)
+* [Deployment](#deployment)
+  - [Download UID2 Private Operator for Azure ZIP File](#download-uid2-private-operator-for-azure-zip-file)
+  * [Create Resource Group](#create-resource-group)
+  * [Complete Key Vault and Managed Identity Setup](#complete-key-vault-and-managed-identity-setup)
+  * [Set Up the VPC Network](#set-up-the-vpc-network)
+  * [Complete the UID2 Private Operator Setup](#complete-the-uid2-private-operator-setup)
+  * [Set Up the Gateway Load Balancer](#set-up-the-gateway-load-balancer)
+* [Running the Health Check](#running-the-health-check)
+* [Upgrading](#upgrading)
+ -->
 
->NOTE: Building and running SGX enclave is powered by [Occlum](https://github.com/occlum/occlum), an open
-source LibOS implementation.
+## Prerequisites
 
-## Build
+Before deploying the UID2 Private Operator for Azure, complete these prerequisite steps:
 
-To build, follow the steps in these sections, in sequence.
+- [Set Up UID2 Operator Account](#set-up-uid2-operator-account)
+- [Install Azure CLI](#install-azure-cli)
+- [Get the Required Azure Permissions](#install-azure-cli)
 
-### Prerequisites
+### Set Up UID2 Operator Account
 
-Make sure the following prerequisites are in place.
+Ask your UID2 contact to register your organization as a UID2 Operator. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
 
- - You must run the build on an Intel SGX enabled machine; tested configuration is Standard_DC8_v2
-   VM running on Azure.
- - Operating system must be Ubuntu 18.04 LTS (recommended Azure Image is of the "Server" variant)
- - Once you have the machine up and running, launch `setup_build_vm.sh` from the `uid2-attestation-azure` repo to install the necessary dependencies.
- - You have sudo permissions during the build.
+When the registration process is complete, you'll receive the following:
 
-### Steps
+- An operator key, exclusive to you, that identifies you with the UID2 service as a Private Operator. During configuration, use this as the value for `OPERATOR_KEY`. This value is both your unique identifier and a password; store it securely and do not share it.
 
-1. Make sure the operator service is built using the `azure` profile:
+  :::note
+  You'll receive a separate operator key for each deployment environment.
+  :::
 
-```
-# From the root source folder
-export enclave_platform=azure-sgx
-./setup_dependencies.sh
-mvn package -P azure
-```
+- A link to the UID2 Private Operator for Azure GitHub release page. For example: [https://github.com/IABTechLab/uid2-operator/releases/tag/v5.20.39-SNAPSHOT-azure-cc](https://github.com/IABTechLab/uid2-operator/releases/tag/v5.20.39-SNAPSHOT-azure-cc).
```

Comment:
Can we change the link to: https://github.com/IABTechLab/uid2-operator/releases/tag/v5.21.5-68a47aec9f-azure-cc

Reply:
Done, thx.